app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/launcher/LauncherActivity.kt

```diff
@@ -0,0 +1,50 @@
+package org.mifos.mobile.cn.ui.mifos.launcher
+
+import android.content.Intent
+import android.os.Bundle
+import android.os.Handler
+import org.mifos.mobile.cn.R
+import org.mifos.mobile.cn.ui.base.MifosBaseActivity
+import org.mifos.mobile.cn.ui.mifos.DashboardActivity
+import org.mifos.mobile.cn.ui.utils.ConstantKeys
+import javax.inject.Inject
+
+/**
+ * @author Manish Kumar
+ * On 31/05/18.
+ */
+
+class LauncherActivity:MifosBaseActivity(),LauncherContract.View {
+
+
+    @Inject
+    lateinit var launcherPresenter: LauncherPresenter
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_launcher)
+        activityComponent.inject(this)
+        launcherPresenter.attachView(this)
```

Comment:
We don't need present and contract in this case because we don't have any logic to write in the presenter. only PrefrenceHelper we would need here.